openssl/src/ssl/mod.rs

```diff
@@ -774,12 +774,12 @@ impl SslContextBuilder {
     /// curve.
     ///
     /// Requires the `v102` feature and OpenSSL 1.0.2.
-    #[cfg(all(feature = "v102", ossl102))]
+    #[cfg(all(feature = "v102", any(ossl102, libressl)))]
```

Comment:
It seems like we'll want to add a separate set of Cargo features for libressl I think.